setup.py

```diff
@@ -189,6 +189,10 @@ def package_files(prefix: str, directory: str, extension: str):
     'apache-libcloud>=3.3.1,<4',
 ]
 
+extra_deps['oci'] = [
+    'oci>=oci-2.88.2',
+]
```

Comment:
```suggestion
extra_deps['oci'] = [
    'oci>=oci-2.88.2,<oci-3.0.0',
]
```
@eracah  can you confirm this new formatting works? why is there version naming weird and prefixed with oci.... In either case, we should pin the version

Reply:
oh good catch. no it shouldn't have oci in the version. fixed